domains.txt

```diff
@@ -394,6 +439,8 @@ linuxrecruit.aeml3.co.uk
 linuxrecruit.co.uk
 logansinclair.com
 logicrec.com
+logikk.co.uk
+logistixrec.co.uk
```

Comment:
Logistix Recruitment Ltd was founded in 1978 as an IT specialist recruitment consultancy. We have a long and successful history of placing IT prof